Examples/timberman.md

```diff
@@ -0,0 +1,72 @@
+# Timberman
+
+**Playstore Link:**  [Timberman](https://play.google.com/store/apps/details?id=com.dm.timber&hl=en)
+
```

Comment:
Images missing. Add images to the /Images folder and use them here. See this as an example: https://github.com/GameAutomators/eBook-Source/blob/master/Examples/05-ready-steady-bang.md